### PR TITLE
Localizing MigrateCommand and fixing one string for dotnet test.

### PR DESCRIPTION
**Pull request by livarcocc**
Fixes https://github.com/dotnet/cli/issues/5416.

@piotroko @piotrpMSFT @jonsequitur @krwq @jgoshi @DustinCampbell

### CODE REVIEW

File: src/dotnet/commands/dotnet-migrate/MigrateCommand.cs [hunk 1]
```diff
@@ -281,17 +281,25 @@ private string GetReportSummary(MigrationReport migrationReport)
             StringBuilder sb = new StringBuilder();
 
             sb.AppendLine("Summary");
-            sb.AppendLine($"Total Projects: {migrationReport.MigratedProjectsCount}");
-            sb.AppendLine($"Succeeded Projects: {migrationReport.SucceededProjectsCount}");
-            sb.AppendLine($"Failed Projects: {migrationReport.FailedProjectsCount}");
+            sb.AppendLine(
```
Comment (piotrpMSFT) [inline src/dotnet/commands/dotnet-migrate/MigrateCommand.cs]: We should add a StringBuilder.AppendLineFormat [naming?] extension method. This pattern is too common in our repo these days... not this PR, of course :)

Comment (jonsequitur) [inline src/dotnet/commands/dotnet-migrate/MigrateCommand.cs]: Instead, I'd suggest representing localizable strings as functions with parameters for interpolated values, e.g. 

```
sb.Append(LocalizableStrings.MigrationReportTotalProjects(
    migrationReport.MigratedProjectsCount));
```

Comment (jonsequitur) [inline src/dotnet/commands/dotnet-migrate/MigrateCommand.cs]: Also, not in this PR. :)  I was planning to this up as part of the resx codegen.

**Comment by livarcocc** [comment]
@dotnet-bot Test OSX x64 Release Build

**Comment by livarcocc** [comment]
@dotnet-bot Test Windows_NT x64 Release Build

**Pull request merged**